app/src/main/res/values-sw360dp/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_podcast">About Podcast</string>
```

Comment:
We define strings in another file, should be somewhere in the ui/i18n folder. Also, I think we should re-use the string that is shown on the "podcast info" page (pressing info icon). If you think "about" is better than "description", the other screen should be changed accordingly to be consistent